src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -121,11 +120,20 @@ public LocalCall(String functionName, Optional<List<?>> arg,
      * @param client SaltClient instance
      * @param target the target for the function
      * @param auth authentication credentials to use
+     * @param batch parameter for enabling and configuring batching
      * @return information about the scheduled job
      */
-    public CompletionStage<LocalAsyncResult<R>> callAsync(final SaltClient client, Target<?> target, AuthMethod auth) {
+    public CompletionStage<LocalAsyncResult<R>> callAsync(final SaltClient client, Target<?> target, AuthMethod auth,
+            Optional<Batch> batch) {
```

Comment:
Nitpick: It would be nice to have an homogeneous interface for calling "callAsync" and "callSync" in batch mode. This requires an Optional<Batch>, while this https://github.com/SUSE/salt-netapi-client/blob/master/src/main/java/com/suse/salt/netapi/calls/LocalCall.java#L281 requires just a "Batch"

Reply:
ah thanks almost fotgot about this again. I will add it tomorrow. :+1:

Reply:
done